native/src/utils/getNavigationDeepLinks.ts

```diff
@@ -1,13 +1,23 @@
-import type { Position } from 'geojson'
 import { Platform } from 'react-native'
 
+import { LocationModel } from 'api-client/src'
+
 /**
  * Depending on the platform this function returns the correct deeplink for navigation app
  * @param address{string}: location address
- * @param coordinates{number[]}: latitude,longitude
+ * @param location{number[]}: latitude,longitude
```

Comment:
This type is not correct.
I'd move all the checks here and just pass the whole `LocationModel` to `getNavigationDeepLinks`